### DISCUSSION TITLE
I can't run Hydrogen. No kernel for grammar Python Found?

### DISCUSSION THREAD

**Discussion by teddybugger** [atom/atom#23569] [Q&A]
Disclaimer: I am sure this has been answered, and I Googled but none of them seem to work unless I missed something. 

I installed Hydrogen, hydrogen-run, and hydrogen-python packages so far. When I run anything, it just says:

No kernel for grammar Python found

Check that the language for this file is set in Atom, that you have a Jupyter kernel installed for it, and that you have configured the language mapping in Hydrogen preferences.

To detect your current Python install you will need to run:

python -m pip install ipykernel
python -m ipykernel install --user

The command above didn't do anything (same error message when I run it). I tried running commands from this link but no luck:

https://nteract.io/kernels

What am I doing wrong?

**Comment by faruqsandi** [comment]
Hello. I wrote https://stackoverflow.com/questions/64711663/setting-up-hydrogen-and-atom-with-anaconda-on-windows/69746725#69746725 before. Maybe you can start there.
.
Also, can you please, attach the output of "jupyter kernelspec list"?
